src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationDynamicAction.swift

```diff
@@ -65,4 +65,11 @@ extension DynamicAction {
 			viewController.navigationController?.pushViewController(toViewController, animated: true)
 		}
 	}
+	
+	static func pushErrorLogsCoordinator() -> Self {
```

Comment:
Are we really pushing in this function, what I can see you are doing that in the end of start function in the coordinator, so probably changing the function name?

Reply:
all the actions in the informationViewController are navigation action so it make since to call the function push ifeven if the pushing happens inside the coordinator